### ISSUE TITLE
python bwrap: Creating new namespace failed

### ISSUE THREAD

**Issue by alex-fu27** [olynch/scientific-fhs#18]
With my home-manager config:
```nix
programs.scientific-fhs = {
enable = true;
enableNVIDIA = false;
juliaVersions = [ {
	version = "1.10.1";
	default = true;
} ];
};
```
When I try to run python, I get:
```terminal
$ python3
bwrap: Creating new namespace failed: nesting depth or /proc/sys/user/max_*_namespaces exceeded (ENOSPC)
```

Julia works fine, on the other hand.

The possible explanation which comes to my mind is that Python might be calling another `python` from `~/.nix-profile/bin/python`, which would spawn a nested bwrap.

**Comment by fivegrant** [comment]
what happens if you run it in a `nix shell nixpkgs#python3`? i encounter this issue as well but i also don't have a system-wide installation of python. it works fine though when i build an application or shell.

**Comment by alex-fu27** [comment]
No, doesn't happen with `nix shell nixpkgs#python3`. But I found where the recursion comes from:
If you call `python3` from `scientific-fhs` (`~/.nix-profile/bin/python3`), then this will execute `bwrap` with a `/nix/store/*-python3-init`, which is a shell script that sets up an `/etc/ld.so.conf` file and finally calls `exec python3 "$@` which resolves to `~/.nix-profile/bin/python3` again, leading to the recursion.
